site/src/Librecores/ProjectRepoBundle/Form/Type/ProjectType.php

```diff
@@ -29,6 +29,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('projectUrl', UrlType::class, array('label' => 'Project URL', 'required' => false))
             ->add('issueTracker', UrlType::class, array('label' => 'Issue/Bug Tracker URL', 'required' => false))
             ->add('sourceRepo', SourceRepoType::class)
+            ->add('classifications', ProjectClassificationType::class)
```

Comment:
You are representing a collection of `ProjectClassification`. This type should be `CollectionType`. This is causing the error here.

```php
$builder->add('tags', CollectionType::class, array(
            'entry_type' => ProjectClassificationType::class,
        ));
```